lib/form/paragraph3.flow

```diff
@@ -374,14 +376,16 @@ renderParagraphEx(
 			default: false;
 		};
 		lastChr = strsubsmart(getWordText(lastElement(group.words, emptyParaWord)), -1, 0);
-		lastChrIsMath = lastChr != "" && strContains("+-=×/", lastChr);
+		firstChr = strsubsmart(getWordText(word), 0, 1);
```

Comment:
check keepFormTogetherOrWithMark, maybe we should move isNonBreakableSpace logic there